api/post.go

```diff
@@ -470,11 +473,11 @@ func getOpenGraphMetadata(c *Context, w http.ResponseWriter, r *http.Request) {
 	og := app.GetOpenGraphMetadata(url)
 
 	ogJSON, err := og.ToJSON()
+	openGraphDataCache.AddWithExpiresInSecs(props["url"], ogJSON, openGraphDataCacheExpireTime) // Cache would expire after 1 week
```

Comment:
@crspeller Okay, I'll revert it back to 1hr since you insist, but most apps/services on the market today will persistent the preview data(e.g.: WhatsApp/Telegram/Instapaper/Pocket), they wouldn't resend any request unless user triggering manually. Invalid cache every hour here makes no sense to me.